### ISSUE TITLE
Document running Specter as a daemon

### ISSUE THREAD

**Issue by stepansnigirev** [cryptoadvance/specter-desktop#630]
`--daemon` is broken, maybe we want to get rid of it instead of fixing it.
There are much better ways to make sure Specter is running and also started on system boot.
I started documenting usage of systemd in https://github.com/cryptoadvance/specter-desktop/pull/621, but we need similar manual for Mac (launchd?) and docker.
@nolimit1 made a great [docker image](https://github.com/lncm/docker-specter-desktop) for Specter, I think it makes sense to mention it in the docs somewhere as well.

**Comment by PaarthAgarwal** [comment]
Do you mean that https://docs.specter.solutions/desktop/daemon/ this section of the docs needs to be deleted?

**Comment by PaarthAgarwal** [comment]
> Do you mean that https://docs.specter.solutions/desktop/daemon/ this section of the docs needs to be deleted?

@moneymanolis any thought on this?

**Comment by moneymanolis** [comment]
@k9ert knows better what is going on here, pretty old issue

**Comment by k9ert** [comment]
Yes, that document would be a good fit to mention docker as well. However, i'm not sure whether there is 
It should mention:
* we're not formally responsible for the docker image but we're very happy that @nol1m1t is managing them here: https://registry.hub.docker.com/r/lncm/specter-desktop/
* https://github.com/lncm/ should be mentioned
* maybe some examples on how to get going with docker
* also mentioning the daemon functionality: https://linuxconfig.org/how-to-start-a-docker-container-as-daemon-process
* However, the above "as-daemon-process" is only 10% of the way. Usually you want that it's starting automatically on startup. So maybe we can provide something based on this approach: https://jugmac00.github.io/blog/how-to-run-a-dockerized-service-via-systemd/
